### PR TITLE
Use domain library in template-manager

### PR DESCRIPTION
**Pull request by stefanerwinmayer**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,12 +13,13 @@ credentials += {
 
 resolvers ++= Seq(
   Resolver.bintrayRepo("ovotech", "maven"),
-  Resolver.bintrayRepo("ovotech", "maven-private")
+  Resolver.bintrayRepo("ovotech", "maven-private"),
+  "confluent-release" at "http://packages.confluent.io/maven/"
```
Comment (ducanhkhu-ovo) [inline build.sbt]: why do you add this on all PRs?

Comment (stefanerwinmayer) [reply build.sbt]: I did not add this on all PRs but this repos is needed for a dependency where ever I have added it.

Comment (stefanerwinmayer) [reply build.sbt]: Example failure:
https://circleci.com/gh/ovotech/comms-template-manager/305

Comment (ducanhkhu-ovo) [reply build.sbt]: same problem as I mentioned in `e2e` repo, kafka dependencies should be in `domain-kafka` module instead of `domain-core`

**Pull request merged**